_articles/tr/building-community.md

```diff
@@ -31,22 +31,22 @@ Topluluğunuzu oluştururken huninin tepesindeki birinin (potansiyel bir kullan
 Belgelerinizle başlayın:
 
 * **Birinin projenizi kullanmasını kolaylaştırın.** [Dostça bir README](../starting-a-project/#readme-yazma) ve sade kod örnekleri, projenize ulaşan herkesin başlamasını kolaylaştıracaktır.
-* [CONTRIBUTING dosyanızı](../starting-a-project/#katkıda-bulunma-rehberinizi-yazmak) kullanarak ve sorun listenizi güncel tutarak **nasıl katkıda bulunulabileceğini açıkça belirtin**.
+* [CONTRIBUTING dosyanızı](../starting-a-project/#katk%C4%B1da-bulunma-rehberinizi-yazmak) kullanarak ve sorun listenizi güncel tutarak **nasıl katkıda bulunulabileceğini açıkça belirtin**.
```

Comment:
Both version is working. The encoding is done by the translation interface that I have used. I checked and saw that it is working so did not change.

Reply:
Hmm, that's odd, the changed fragment in best-practices.md went in the opposite direction (escaped->non-escaped) in this PR!

In general I'd prefer a smaller diff if there's no functional difference, but not a big deal since both work.